src/assets/styles/layout.scss

```diff
@@ -11,17 +11,20 @@ body,
 
 .layout-main {
   flex: 1;
-  display: flex;
```

Comment:
I think there are a number of things that depend on this primary container being `flex` -- after merging in master would be good to check that layout things are still ok, also possibly the loading spinner would be affected IIRC